cmd/transportRequestUploadRFC.go

```diff
@@ -81,5 +86,15 @@ func runTransportRequestUploadRFC(config *transportRequestUploadRFCOptions, acti
 	action.WithTransportRequestID(config.TransportRequestID)
 	action.WithApplicationURL(config.ApplicationURL)
 
-	return action.Perform(cmd)
+	commonPipelineEnvironment.custom.transportRequestID = config.TransportRequestID
```

Comment:
Why do we now need to set this variable in the commonPipelineEnvironment? That makes it available throughout the pipeline, right?

Reply:
yes, the idea is to propagate the id, used by the step. the id can come in by para, config or cpenv. Output is always cpenv. 
So succeeding steps could consume the id from cpe. Like a transport request release step.